src/dump_analysis.cpp

```diff
@@ -732,6 +732,27 @@ static void anal_dump_pointer_attrs(AnalDumpCtx *ctx, ZigType *ty) {
     anal_dump_type_ref(ctx, ty->data.pointer.child_type);
 }
 
+static void anal_dump_struct_field(AnalDumpCtx *ctx, const TypeStructField *struct_field) {
+    JsonWriter *jw = &ctx->jw;
+
+    jw_begin_object(jw);
+
+    jw_object_field(jw, "name");
+    jw_string(jw, buf_ptr(struct_field->name));
+
+    jw_object_field(jw, "type");
+    if (struct_field->type_entry != nullptr) {
+        anal_dump_type_ref(ctx, struct_field->type_entry);
+    } else {
+        jw_null(jw);
```

Comment:
was there a case where this happened? just curious, what struct was it?

Reply:
This check was actually unnecessary.